spark/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java

```diff
@@ -190,9 +223,24 @@ private Map setupPySparkEnv() throws IOException{
       }
     }
 
+    LOGGER.debug("PYTHONPATH: " + env.get("PYTHONPATH"));
     return env;
   }
 
+  // Run python shell
+  // Choose python in the order of
+  // PYSPARK_DRIVER_PYTHON > PYSPARK_PYTHON > zeppelin.pyspark.python
+  public static String getPythonExec(Properties properties) {
+    String pythonExec = properties.getProperty("zeppelin.pyspark.python", "python");
```

Comment:
what if `zeppelin.python` is set?

Reply:
`zeppelin.python` is only for python interpreter.  And `zeppelin.pyspark.python` is for pyspark interpreter.